mlir/test/Dialect/AMX/invalid.mlir

```diff
@@ -4,45 +4,45 @@
 
 func.func @rowheight() {
   // expected-error@+1 {{'amx.tile_zero' op bad row height: 17}}
-  %0 = amx.tile_zero : vector<17x16xbf16>
+  %0 = amx.tile_zero : <17x16xbf16>
```

Comment:
this is a weird syntax. Why not `x86_amx<17x6xbf16>`?

Reply:
That looked odd to me as well. The reason appears to be in printers/parsers generated for operations. Here we have `amx::TileType` type that uses `amx.tile` mnemonic and is printed as `amx.tile<17x6xbf16>`. But operation's printers use `printStrippedAttrOrType` to print argument/result types. I suppose that's because they are known to always be `amx::TileType`. So, e.g. for block arguments we have `amx.tile<17x6xbf16>` but in AMX operation we have simply `<17x6xbf16>`.

Reply:
OK. I'll search for a way to fix those printers.

Reply:
Used `qualified` to force type mnemonic for all op types.